buildlib/pr/codestyle.yml

```diff
@@ -70,3 +70,25 @@ jobs:
               azure_complete_with_issues "Code is not formatted according to the code style, see $url for more info."
           fi
         condition: eq(variables['Build.Reason'], 'PullRequest')
+
+  - job: codespell
+    displayName: codespell check
+    pool:
+      name: MLNX
+      demands:
+      - ucx_docker -equals yes
+    container: fedora
+    steps:
+      - checkout: self
+        clean: true
+        fetchDepth: 100
+        retryCountOnTaskFailure: 5
+
+      - bash: |
```

Comment:
Cosmetic: It's advised to set a `displayName` per step.

Reply:
Done

Reply:
is it really needed. I see no other job in this file do that

Reply:
```
arivkin@mtr-vdi-007:ucx (test_PR) $ grep -r displayName * | wc -l     
101
```
Some do it right.
I would add displayName to other steps in this file as well.

Reply:
AFAIU it is a cosmetic change, so it is not strongly required, do you propose to remove it?